docs/concepts/project.md

```diff
@@ -92,10 +92,9 @@ Starting from the [Create Entity](https://app.prophecy.io/metadata/create) page,
 
 ![Create Entity page](img/create_entity.png)
 
-In the `Create Project` pane you can set the name, output language (Scala or Python) and which team the project belongs to. It is strongly recommended that you connect to your Git repository
-to ensure that there is a secure copy of the code that you have direct access to.
+In the `Create Project` pane you can set the name, output language (Scala or Python) and which team the project belongs to. Add [Git credentials](./../metadata/Git) in [Settings](https://app.prophecy.io/metadata/settings) or connect new Git credentials as below.
```

Comment:
Can we add a small note here that Language here is different from the language within Expression boxes.

Reply:
![image](https://user-images.githubusercontent.com/20563764/221651654-137d65fc-3412-4882-80f3-5a79100b6fdc.png)